java/src/jmri/jmrit/operations/rollingstock/cars/tools/ImportCars.java

```diff
@@ -191,29 +191,35 @@ public void run() {
                 carComment = "";
 
                 if (inputLine.length > base + CAR_WEIGHT) {
-                    carWeight = inputLine[base + CAR_WEIGHT];
+                    carWeight = inputLine[base + CAR_WEIGHT].trim();
                 }
                 if (inputLine.length > base + CAR_COLOR) {
-                    carColor = inputLine[base + CAR_COLOR];
+                    carColor = inputLine[base + CAR_COLOR].trim();
                 }
 
                 log.debug("Checking car number ({}) road ({}) type ({}) length ({}) weight ({}) color ({})", carNumber,
                         carRoad, carType, carLength, carWeight, carColor); // NOI18N
 
-                if (carNumber.trim().isEmpty()) {
+                if (carNumber.isEmpty()) {
                     log.info("Import line {} missing car number", lineNum);
                     break;
                 }
-                if (carRoad.trim().isEmpty()) {
+                if (carRoad.isEmpty()) {
                     log.info("Import line {} missing car road", lineNum);
                     break;
                 }
-                if (carType.trim().isEmpty()) {
+                if (carType.isEmpty()) {
                     log.info("Import line {} missing car type", lineNum);
+                    JOptionPane.showMessageDialog(null, MessageFormat.format(
+                            Bundle.getMessage("CarTypeNotSpecified"), new Object[]{(carRoad + " " + carNumber)}),
+                            Bundle.getMessage("CarTypeMissing"), JOptionPane.ERROR_MESSAGE);
```

Comment:
Is there a risk that this error will happen multiple times during import? For example that the user has forgot to set the type for 20 cars? If so, it might be good to either:
- Add the option "Don't show again" to the dialog
- Collect the errors and show the error dialog once with all the errors

Reply:
If the type field is missing the import aborts showing the user the import line is a status window.  The entire code:

                if (carType.isEmpty()) {
                    log.info("Import line {} missing car type", lineNum);
                    JOptionPane.showMessageDialog(null, MessageFormat.format(
                            Bundle.getMessage("CarTypeNotSpecified"), new Object[]{(carRoad + " " + carNumber)}),
                            Bundle.getMessage("CarTypeMissing"), JOptionPane.ERROR_MESSAGE);
                    break;
                }